src/classes/json_data.py

```diff
@@ -309,6 +304,13 @@ def replace_string_to_relative(self, match):
             # Convert path to the correct relative path (based on the existing folder)
             orig_abs_path = os.path.abspath(path)
 
+            # Determine windows drives that the project and file are on
+            project_win_drive = os.path.splitdrive(get_app().project.current_filepath)[0]
+            file_win_drive = os.path.splitdrive(path)[0]
+            if file_win_drive != project_win_drive:
+                # If the file is on different drive. Don't abbreviate the path.
+                return orig_abs_path
```

Comment:
I just realized (sorry), we **could** still do the path string cleanup here, i.e.
```suggestion
                return orig_abs_path.replace("\\", "/")
```

Reply:
(I've never been super big on the path munging, as it's dangerous and really shouldn't be necessary... but if we're going to do it, the safest place is here, not after the project data is already embedded in a giant JSON `str` object.)

Reply:
Oh, duh! Also, this should still be returning a JSON string-ified version of the path. So, actually:
```suggestion
                return f"{key}: {orig_abs_path}".replace("\\", "/")
```
Sorry, I totally spaced on what the actual purpose of this function was.

Reply:
> (I've never been super big on the path munging, as it's dangerous and really shouldn't be necessary... but if we're going to do it, the safest place is here, not after the project data is already embedded in a giant JSON `str` object.)

Sorry, 'munging'? Is this because windows's `\`'s are usually an escape character, and would cause problems in reading the json?

Reply:
I do see that all the other ends of this function return a key-value, not a plain path. My bad

Reply:
@JacksonRG 

Yes and no. The backslashes never actually caused any problems for `json.dumps`/`json.loads`; it knows to double them and deals fine. Our own string-manipulating code — which at the time was running over the _entire_ string object containing the **full** project JSON! — would get confused by them, was the issue.